src/main/java/at/hannibal2/skyhanni/features/misc/reminders/ReminderManager.kt

```diff
@@ -0,0 +1,256 @@
+package at.hannibal2.skyhanni.features.misc.reminders
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.SecondPassedEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.Base62
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import at.hannibal2.skyhanni.utils.TimeUtils
+import at.hannibal2.skyhanni.utils.TimeUtils.format
+import at.hannibal2.skyhanni.utils.TimeUtils.minutes
+import at.hannibal2.skyhanni.utils.chat.Text
+import at.hannibal2.skyhanni.utils.chat.Text.asComponent
+import at.hannibal2.skyhanni.utils.chat.Text.center
+import at.hannibal2.skyhanni.utils.chat.Text.command
+import at.hannibal2.skyhanni.utils.chat.Text.fitToChat
+import at.hannibal2.skyhanni.utils.chat.Text.hover
+import at.hannibal2.skyhanni.utils.chat.Text.send
+import at.hannibal2.skyhanni.utils.chat.Text.style
+import at.hannibal2.skyhanni.utils.chat.Text.suggest
+import at.hannibal2.skyhanni.utils.chat.Text.wrap
+import net.minecraft.util.EnumChatFormatting
+import net.minecraft.util.IChatComponent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.seconds
+
+@SkyHanniModule
+object ReminderManager {
+
+    private const val REMINDERS_PER_PAGE = 10
+
+    // Random numbers chosen, this will be used to delete the old list and action messages
+    private const val REMINDERS_LIST_ID = -546745
+    private const val REMINDERS_ACTION_ID = -546746
+    private const val REMINDERS_MESSAGE_ID = -546747
+
+    private val storage get() = SkyHanniMod.feature.storage.reminders
+    private val config get() = SkyHanniMod.feature.misc.reminders
+
+    private var listPage = 1
+
+    private fun newId(): String {
```

Comment:
Can't we simply use UUID.Random().toString() here? The Base62 and storage approach feels like overkill

Reply:
yeah I wrote it with users also being able to manually type it out in mind but since the change to a nicer chat interface it can just be changed to a random of anything really